templates/redis/bin/common.sh

```diff
@@ -0,0 +1,85 @@
+# Environment variable common to all scripts
+APISERVER=https://kubernetes.default.svc
+SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+TOKEN=$(cat ${SERVICEACCOUNT}/token)
+CACERT=${SERVICEACCOUNT}/ca.crt
+
+TIMEOUT=3
+
+PODNAME=$(echo $HOSTNAME | cut -d. -f1,2)
+PODIP=$(grep "${PODNAME}" /etc/hosts | cut -d$'\t' -f1)
+# redis-redis-0.redis-redis.openstack.svc.cluster.local
+PODFQDN=$(getent hosts "${PODIP}" | awk '{print $2}')
+# redis-redis.openstack.svc.cluster.local
+SVCFQDN=$(echo "${PODFQDN}" | sed 's/^[^\.]*.//')
+
+function log() {
+    echo "$(date +%F_%H_%M_%S) $*"
+}
+
+function log_error() {
+    echo "$(date +%F_%H_%M_%S) ERROR: $*"
+}
+
+function generate_configs() {
+    # Copying config files except template files
+    mkdir -p /var/lib/kolla/config_files/generated
+    tar -C /var/lib/kolla/config_files --exclude '..*' --exclude '*.in' -h -c src | tar -C /var/lib/kolla/config_files/generated -x --strip=1
+    # Generating config files from templates
+    cd /var/lib/kolla/config_files/src
+    for cfg in $(find -L * -name '*.conf.in'); do
+        log "Generating config file from template $PWD/${cfg}"
+        sed -e "s/{ PODNAME }/${PODNAME}/" -e "s/{ PODIP }/${PODIP}/" -e "s/{ PODFQDN }/${PODFQDN}/" "${cfg}" > "/var/lib/kolla/config_files/generated/${cfg%.in}"
```

Comment:
not seeing the PODIP in the templates. seems to be only the PODFQDN

Reply:
Ack, removed as we don't need it. Thanks.